package.json

```diff
@@ -39,7 +39,7 @@
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.3.3",
-    "@openmrs/react-root-decorator": "^1.0.0",
+    "@openmrs/react-root-decorator": "file:../openmrs-react-root-decorator",
```

Comment:
We should not push a package json that installs from file instead of from npm. If you intend this to be a draft/wip PR, please [mark it in Github as draft](https://github.blog/2019-02-14-introducing-draft-pull-requests/) (at least for future PRs).

Reply:
Oh sweet! I didn’t know about that feature! Yes this is a WIP.